extensions/cdc-debezium/src/test/java/com/hazelcast/jet/cdc/DebeziumCdcIntegrationTest.java

```diff
@@ -27,6 +27,7 @@
 import com.hazelcast.jet.pipeline.StreamSource;
 import com.hazelcast.test.HazelcastSerialClassRunner;
 import com.hazelcast.test.annotation.NightlyTest;
+import io.debezium.connector.mysql.MySqlConnector;
```

Comment:
This leaks debezium API to our API. We should avoid that.

Reply:
It's test only. I've changed the Source method to have only Class<?> and type check is done in the method code